Polygon_mesh_processing/examples/Polygon_mesh_processing/isotropic_remeshing_example.cpp

```diff
@@ -35,14 +37,20 @@ int main(int argc, char* argv[])
 {
   const char* filename = (argc > 1) ? argv[1] : "data/pig.off";
 
+  CGAL::Timer t;
+  t.start();
+
   Mesh mesh;
   if(!PMP::read_polygon_mesh(filename, mesh) || !CGAL::is_triangle_mesh(mesh))
   {
     std::cerr << "Invalid input." << std::endl;
     return 1;
   }
 
-  double target_edge_length = 0.04;
+  std::cout << "Read: " << t.time() << " sec" << std::endl;
+  t.reset();
+
+  double target_edge_length = (argc > 2) ? std::stod(std::string(argv[2])) : 1.0;
```

Comment:
Any reason to change the previous default?

Reply:
By accident.